src/components/MultiSelect/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, {useState} from 'react';
+import './styles.scss';
+import IconTooltip from '../IconTooltip';
+import {IconButton, TooltipProps} from '@mui/material';
+import Icon from '../Icon/Icon';
+import PillList, {PillListItem} from '../PillList';
+
+export type MultiSelectProps<T> = {
+  className?: string;
+  fullWidth?: boolean;
+  helperText?: string;
+  id?: string;
+  label?: string;
+  onAdd: (item: T) => void;
+  onRemove: (item: T) => void;
+  options: Map<T, string>;
+  placeHolder?: string;
+  selectedOptions: T[];
+  tooltipTitle?: TooltipProps['title'];
+};
+
+export default function MultiSelect<T>(props: MultiSelectProps<T>): JSX.Element {
+  const {
+    className,
+    fullWidth,
+    helperText,
+    id,
+    label,
+    placeHolder,
+    onAdd,
+    onRemove,
+    options,
+    selectedOptions,
+    tooltipTitle,
+  } = props;
+
+  const [openedOptions, setOpenedOptions] = useState(false);
+
+  const toggleOptions = () => {
+    setOpenedOptions((isOpen) => !isOpen);
+  };
+
+  const unSelectedOptions = Array.from<T>(options.keys()).filter((key: T) => !selectedOptions.includes(key));
```

Comment:
Should it be unselectedOptions? without the capital "S"

Reply:
that's probably nicer spelling, yes